### DISCUSSION TITLE
[2024-01-29] Team GUI Demo Day

### DISCUSSION THREAD

**Discussion by farmaazon** [enso-org/enso#8886] [Weekly Demo Day]
@farmaazon 

Mostly bugfixing:
* #8874
* #8876
* #8859
* #8803

[Screencast from 2024-01-19 09-06-48.webm](https://github.com/enso-org/enso/assets/3919101/6df28a9c-51b3-4f98-be10-a35275eac800)

@Frizi 
* #8860

**Comment by vitvakatu** [comment]
- https://github.com/enso-org/enso/pull/8873
- https://github.com/enso-org/enso/pull/8844
- https://github.com/enso-org/enso/pull/8837 got merged


https://github.com/enso-org/enso/assets/6566674/c68f1df4-285e-413b-b05e-13394dffe3bd


https://github.com/enso-org/enso/assets/6566674/966576ec-6507-401c-98d3-bd71c2ffc6b2

**Comment by MichaelMauderer** [comment]
Expanding e2e tests
* added tests for node typing ( https://github.com/enso-org/enso/pull/8856 )
* working on adding tests for edge rendering followed by edge interactions

**Comment by kazcw** [comment]
- #8237:
  - #8851
  - #8880 (In code review / updating e2e test support)

**Comment by mwu-tow** [comment]
Last week:
* Fixed a disk space leak on sef-hosted runners: https://github.com/enso-org/ci/pull/3
* The PR that enables native aarch64 macOS builds in releases is now ready for review: https://github.com/enso-org/enso/pull/8864.
  * The experimental nightly build from the branch is available [here](https://github.com/enso-org/enso/releases/tag/2024.1.1-nightly.2024.1.29.1).
  * There is intermittent failure with code signing with an internal error reported. It needs further investigation. Might be due to runner configuration, rather than the PR.
* Investigating CI stability issues.
  * Nightlies failing way too often — I am manually rerunning the failed jobs.
  * Checking other transient failures.

**Comment by hubertp** [comment]
Related backend work:
- visualizations not loading or flickering #8853
